src/directives/index.js

```diff
@@ -0,0 +1,344 @@
+/**
+ * ORIGINAL SOURCE: src/app/gui/vue/vue.directives.js@v3.6
+ */
+
+import ApplicationState from 'core/applicationstate';
+import vDisabled from 'directives/v-disabled';
+import vChecked from 'directives/v-checked';
+import vSelectedFirst from 'directives/v-selected-first';
+import vSelect2 from 'directives/v-select2';
+
+const {t, tPlugin} = require('core/i18n/i18n.service');
+const { uniqueId, toRawType } = require('core/utils/utils');
+
+const GlobalDirective = {
+  install(Vue) {
+    const vm = new Vue();
+    const directives = {};
+    const createDirectiveObj = ({el, attr}) =>{
+      //create unique id
+      const unique_attr_id = uniqueId();
+      // set new attribute
+      el.setAttribute(attr, unique_attr_id);
+      directives[unique_attr_id] = {};
+      return unique_attr_id;
+    };
+    const setUnwatch = ({id, unwatch})=>{
+      directives[id].unwatch = unwatch;
+    };
+    const unbindWatch = ({attr, el})=>{
+      const unique_attr_id = el.getAttribute(attr);
+      if (unique_attr_id) {
+        directives[unique_attr_id].unwatch();
+        delete directives[unique_attr_id];
+      }
+    };
+    const runHandlerOnUpdate = ({el, attrId, attr, oldValue})=>{
+      const unique_attr_id =  el.getAttribute(attrId);
+      const attr_value = el.getAttribute(attr);
+      (attr_value != null && attr_value !== oldValue) && directives[unique_attr_id].handler({el});
+    };
+    const prePositioni18n = ({el, binding, i18nFnc=t}) => {
+      const innerHTML = el.innerHTML;
+      const position = binding.arg ? binding.arg : 'post';
+      const handlerElement = innerHTML => {
+        const value = binding.value !== null ?  i18nFnc(binding.value) : '';
+        if (position === 'pre') el.innerHTML =  `${value} ${innerHTML}`;
+        else if (position === 'post') el.innerHTML = `${innerHTML} ${value}`;
+      };
+      handlerElement(innerHTML);
+      return vm.$watch(() => ApplicationState.lng, () => handlerElement(innerHTML));
+    };
+
+    Vue.directive("disabled", vDisabled);
+    Vue.directive("checked", vChecked);
+    Vue.directive("selected-first", vSelectedFirst);
+
+    /**
+     * TODO: split and refactor into a separated file (eg. "./directives/v-t-tooltip.js"):
+     */
```

Comment:
@volterra79 does it seem feasible to you?

Reply:
Closed by https://github.com/g3w-suite/g3w-client/pull/174/commits/316b9af7e433a5e153a58a1343da430f2495a772 and https://github.com/g3w-suite/g3w-client/pull/174/commits/f353c8eda6089d8d1a4e13ab846ab4fe27ee6e00